SCAutolib/models/file.py

```diff
@@ -207,7 +293,8 @@ def create(self):
             logger.info(f"{self._conf_file} file exists, loading values")
             logger.info(f"Backing up {self._conf_file}"
                         f"as {self._backup_original}")
-            copy2(self._conf_file, self._backup_original)
```

Comment:
As this is replaced by self.backup method that already prints logs, above line with log can be removed.